.github/workflows/tests.yml

```
@@ -47,6 +47,7 @@ jobs:
                 -DCMAKE_BUILD_TYPE=Release        \
                 -DCMAKE_INSTALL_PREFIX=/usr/local \
                 -DOT_COMM_COVERAGE=ON             \
+                -DOT_COMM_COMMISSIONING_LOG=ON    \
```

```suggestion
                -DOT_COMM_REFERENCE_DEVICE=ON    \
```

OT used to use `CERT_LOG` for certification logs, but later changed to `REFERENCE_DEVICE`.
Should we be consistent with OT and OTBR here?

---

Agree, will make the change accordingly.